pages/company/company.go

```diff
@@ -73,5 +95,5 @@ func Get(ctx aero.Context) error {
 		})
 	}
 
-	return ctx.HTML(components.CompanyPage(company, studioAnime, producedAnime, licensedAnime, closeCompanies, distances, user))
+	return ctx.HTML(components.CompanyPage(company, studioAnime, producedAnime, licensedAnime, closeCompanies, distances, user, overallRating, storyRating, visualsRating, soundtrackRating, overallCount, storyCount, visualsCount, soundtrackCount))
```

Comment:
I think we can use a struct for this to avoid the parameter hell, it possibly even exists already.

Reply:
Oh yeah, I honestly didn't consider using structs. Would have made sense.